src/setters/factors.ts

```diff
@@ -49,6 +49,26 @@ export const setPwrdFactor = (): void => {
     factor.save();
 }
 
+export const updateGTokenFactor = (): void => {
```

Comment:
This function looks redundant.
Depending on the need, I would just call `setGvtFactor()` and/or `setPwrdFactor()`, specially if we change the factor update depending on withdrawal events where we will call only one of them.

Reply:
When do harvest and withdraw, the both factors will be change. we should update them. in the Listener, we only update one is for we attached the factor to transfer event, and there is only one token event.

Reply:
A pwrd withdrawal is affecting the gvt factor?

Reply:
yes, pwrd&gvt's factors are changed at the same time